charts/policy-reporter/charts/monitoring/Chart.yaml

```diff
@@ -3,5 +3,5 @@ name: monitoring
 description: Policy Reporter Monitoring with predefined ServiceMonitor and Grafana Dashboards
 
 type: application
-version: 1.3.0
+version: 1.4.0
```

Comment:
Hey, thank you for this great improvement. To fullfill the linting you will have to update the main Chart as well. You can update the Chart Version to 1.8.0, update the `monitoring` subchart dependency and run `helm dependency update policy-reporter` inside of the `charts` folder.

Reply:
I'm sorry for the missing Contribution Documentation, I will improve this in the near future